src/pages/Components/Notification/previews/notification-danger.preview.scss

```diff
@@ -0,0 +1,11 @@
+@import 'settings-tools/_all-settings';
+@import 'components/_c.notification';
+
+.example {
+  height: 250px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  margin: $mu150;
+}
```

Comment:
Why do you fix the height ?

Reply:
if I don't fix the height, or if I don't take any step to increase the frame height, I'll end up with my content hidden (in height).